src/convertToObject.js

```diff
@@ -7,6 +7,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to make the code more readable.